spec/system/task_spec.rb

```diff
@@ -2,35 +2,41 @@
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    
+    let(:project) {create(:project)}
+    let(:task) {create(:task, project_id: project.id)}
```

Comment:
ブレースの中は1文字開けた方が見やすいですね
```suggestion
    let(:project) { create(:project) }
    let(:task) { create(:task, project_id: project.id) }
```

Reply:
5295244 で修正しました。